src/sagemaker/model.py

```diff
@@ -2152,7 +2148,7 @@ def update_approval_status(self, approval_status, approval_description=None):
         """Update the approval status for the model package
 
         Args:
-            approval_status (str or PipelineVariable): Model Approval Status, values can be
+            approval_status (str): Model Approval Status, values can be
```

Comment:
Why removing PipelineVariable from description. Can't it be pipelineVariable if happening from sagemaker pipeliens?

Reply:
Why removing PipelineVariable from description. Can't it be pipelineVariable if happening from sagemaker pipeliens?